Mathlib/Tactic/Linter/DeprecatedSyntaxLinter.lean

```diff
@@ -52,39 +55,55 @@ register_option linter.style.cases : Bool := {
   descr := "enable the cases linter"
 }
 
+/-- The option `linter.style.admit` of the deprecated syntax linter flags usages of
+the `admit` tactic, which is a synonym for the much more common `sorry`. -/
+register_option linter.style.admit : Bool := {
+  defValue := false
+  descr := "enable the admit linter"
+}
+
 /-- `getDeprecatedSyntax t` returns all usages of deprecated syntax in the input syntax `t`. -/
 partial
 def getDeprecatedSyntax : Syntax → Array (SyntaxNodeKind × Syntax × MessageData)
   | stx@(.node _ kind args) =>
     let rargs := (args.map getDeprecatedSyntax).flatten
-    if ``Lean.Parser.Tactic.refine' == kind then
+    match kind with
+    | ``Lean.Parser.Tactic.refine' =>
       rargs.push (kind, stx,
         "The `refine'` tactic is discouraged: \
          please strongly consider using `refine` or `apply` instead.")
-    else if `Mathlib.Tactic.cases' == kind then
+    | `Mathlib.Tactic.cases' =>
       rargs.push (kind, stx,
         "The `cases'` tactic is discouraged: \
          please strongly consider using `obtain`, `rcases` or `cases` instead.")
-    else rargs
+    | ``Lean.Parser.Tactic.tacticAdmit =>
+      rargs.push (kind, stx,
+        "The `admit` tactic is discouraged: \
+         please strongly consider using the synonymous `sorry` instead.")
+    | _ => rargs
   | _ => default
 
 /-- The deprecated syntax linter flags usages of deprecated syntax and suggests
 replacement syntax.
 
-Currently `refine'` (superseded by `refine`) and `cases'` (superseded by `obtain`, `rcases` and
-`cases`) are flagged. The linter can be turned off for these cases individually with the options
-`linter.style.refine` and `linter.style.cases`.
+Currently the following syntax is flagged. The option to turn off linting for each individual case
```

Comment:
How about this wording?
"For each individual case, linting can be turned on or off separately."
and then word the items like
"* `refine'`, superseded by `refine` and `apply` (controlled by `linter.style.refine`)

Reply:
OK, I have used your suggested wording.